### PR TITLE
WRN-16290 [Agate] Sampler: Merge migrated storybook with controls to develop branch

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [ ] UI test was passed or is not needed
* [ ] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Merge migrated storybook with controls to develop branch

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-16290

### CODE REVIEW

File: samples/sampler/stories/default/ColorPicker.js [hunk 1]
```diff
@@ -23,16 +23,15 @@ export default {
...
+export const _ColorPicker = (args) => {
+	const direction = args['direction'];
 	const colors = prop.presets[
-		select('color palette', Object.keys(prop.presets), StoryOptions, 'Default')
+		args['color palette'] || 'Default'
 	];
-
 	return (
 		<ColorPicker
 			direction={direction}
```
Comment (juwonjeong) [inline samples/sampler/stories/default/ColorPicker.js:33]:
```suggestion
			direction={direction}
```
```suggestion
			direction=args['direction']
```

Comment (juwonjeong) [inline samples/sampler/stories/default/ColorPicker.js:33]:
```suggestion
			direction={direction}
```
```suggestion
			direction=args['direction']
```

File: samples/sampler/stories/default/Panels.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ const SecondPanel = kind({
 	)
 });
 
-const BasicPanels = ({...rest}) => {
+const BasicPanels = ({orientation, ...rest}) => {
```
Comment (juwonjeong) [inline samples/sampler/stories/default/Panels.js:43]:
```suggestion
const BasicPanels = ({orientation, ...rest}) => {
```
```suggestion
const BasicPanels = ({...rest}) => {
```

File: samples/sampler/stories/default/Panels.js [hunk 3]
```diff
@@ -50,27 +50,36 @@ const BasicPanels = ({...rest}) => {
 			{...rest}
 			index={index}
 			onBack={goPrevious}
-			orientation={select('orientation', ['horizontal', 'vertical'], Config)}
+			orientation={orientation}
```
Comment (juwonjeong) [inline samples/sampler/stories/default/Panels.js:53]: Can we delete this line 53 by modifying line 43?

File: samples/sampler/stories/default/ColorPicker.js [hunk 4]
```diff
@@ -41,6 +40,10 @@ export const _ColorPicker = () => {
 	);
 };
 
+select('direction', _ColorPicker, prop.direction, Config, 'right');
+boolean('disabled', _ColorPicker, Config);
+select('color palette', _ColorPicker, Object.keys(prop.presets), StoryOptions, prop.presets['Default']);
```
Comment (juwonjeong) [inline samples/sampler/stories/default/ColorPicker.js:45]:
```suggestion
select('color palette', _ColorPicker, Object.keys(prop.presets), StoryOptions, 'Default');
```

File: samples/sampler/stories/default/Popup.js [hunk 5]
```diff
@@ -25,36 +25,50 @@ export default {
...
+				<div>{args['children']}</div>
 				{buttons}
 			</Popup>
-			Use KNOBS to interact with Popup.
+			Use CONTROLS to interact with Popup.
 		</div>
 	);
 };
 
+select('buttons', _Popup, ['no buttons', '1 button', '2 buttons'], Config, 'no');
```
Comment (juwonjeong) [inline samples/sampler/stories/default/Popup.js:58]: Please fix this line 
```suggestion
select('buttons', _Popup, ['no buttons', '1 button', '2 buttons'], Config, 'no buttons');
```

File: samples/sampler/stories/default/TabbedPanels.js [hunk 6]
```diff
@@ -37,7 +36,7 @@ const I18nTabbedPanelsBase = ({rtl, ...rest}) => {
 				index={panelIndex}
 				onSelect={onSelect} // eslint-disable-line react/jsx-no-bind
 				orientation={orientation}
-				tabPosition={select('tabPosition', ['before', 'after'], Config, 'before')}
+				tabPosition={tabPosition}
 				tabs={[
```
Comment (juwonjeong) [inline samples/sampler/stories/default/TabbedPanels.js:40]: Can we remove tabPosition as it can be included in {...rest} ?

**Comment by juwonjeong** [comment]
Please change placehold.it image service in default/virtualGridList.js as https://github.com/enactjs/sandstone/pull/1185

File: samples/sampler/stories/default/IncrementSlider.js [hunk 7]
```diff
@@ -60,6 +59,22 @@ export const _IncrementSlider = () => {
 	);
 };
 
+select('orientation', _IncrementSlider, ['horizontal', 'vertical'], Config);
```
[the remark this answers is not in the document]
Comment (vladut-clapou-lgp) [reply samples/sampler/stories/default/IncrementSlider.js:62]: you mean by function(bool, number, etc) or control name(activateOnFocus, disabled, etc)?

[the remark this answers is not in the document]
Comment (juwonjeong) [reply samples/sampler/stories/default/IncrementSlider.js:62]: with control names please.

File: samples/sampler/custom-addon/ColorPicker.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+import {ACCENT_DEFAULT_VALUE, ACCENT_PARAM_KEY, HIGHLIGHT_DEFAULT_VALUE} from './constants';
+
+const getDefaultColor = (colorPickerType) => {
+	if (colorPickerType === ACCENT_PARAM_KEY) return ACCENT_DEFAULT_VALUE;
+	return HIGHLIGHT_DEFAULT_VALUE;
+};
+
+const ColorPicker = ({colorPickerType}) => {
+	const [globals, updateGlobals] = useGlobals();
+
```
Comment (juwonjeong) [inline samples/sampler/custom-addon/ColorPicker.js:14]:
```suggestion
	const handleChange = useCallback((ev) => {
		updateGlobals({[colorPickerType]: ev.target.value});
	}, [colorPickerType, updateGlobals]);
```

File: samples/sampler/custom-addon/ColorPicker.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+		});
+	}, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+	return (
+		<input
+			type="color"
+			value={globals[colorPickerType] || getDefaultColor(colorPickerType)}
+			onChange={(e) => { // eslint-disable-line react/jsx-no-bind
+				updateGlobals({[colorPickerType]: e.target.value});
+			}}
```
Comment (juwonjeong) [inline samples/sampler/custom-addon/ColorPicker.js:29]:
```suggestion
			onChange={handleChange}
```

**Comment by juwonjeong** [comment]
Also, please resolve conflicts.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**